src/bin/inx-chronicle/cli.rs

```diff
@@ -196,4 +235,11 @@ pub enum Subcommands {
     /// Generate a JWT token using the available config.
     #[cfg(feature = "api")]
     GenerateJWT,
+    //#[cfg(feature = "analytics")]
```

Comment:
This will probably change once we merge the other PR, right?

Reply:
Yes, that's the idea.